src/index.html

```diff
@@ -20,8 +20,504 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
```

Comment:
Matheus, you could add an empty line here and between other sibling elements in this file, check [Code Styles Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Add%20empty%20lines%20between%20multiline%20sibling%20blocks%20of%20HTML)